package/yast2-bootloader.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Nov 20 10:19:58 UTC 2025 - Stefan Schubert <schubi@suse.de>
+
+- Added requirement dbus-1-tools because /usr/bin/dbus-uuidgen is
+  needed by BLS (bnc#1253724).
```

Comment:
sigh, if BLS will use common uuidgen from util-linux it will make life easier. I check that agama already have it on its TW iso. Can you just double check that it is available also on SLE-16? as it will be added also to agama iso as dependency, even when agama do not use bls bootloader yet.